### DISCUSSION TITLE
Create new app with quasar create <folder_name> OR yarn create quasar

### DISCUSSION THREAD

**Discussion by perottosson** [quasarframework/quasar#14319] [General - Components / Directives / etc] [answered]
So far I have created my Quasar apps with `quasar create <folder_name>` and are running the app with `quasar dev`. I have so far used Vuex as state manager but will switch over to pinia which is recommended. However I wont get the option of creating an app with pinia running `quasar create`. 

Is this depraciated? Shall I use `yarn create quasar` instead? And running it with `yarn dev` instead

**Comment by metalsadman** [comment, marked as answer]
yes that's the new command, `quasar dev` is still the command for opening the dev server.
